package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "react-accessible-dropdown-menu-hook",
-	"version": "3.1.0",
+	"version": "3.1.1",
```

On its own, this PR would only need to bump the patch version. But, in combination with #290, I think it's worthy of bumping the minor version instead:

```suggestion
	"version": "3.2.0",
```